app/assets/stylesheets/components/all.scss

```diff
@@ -20,7 +20,6 @@
 @import 'personal-key';
 @import 'position';
 @import 'tooltip';
-@import 'icon';
```

Comment:
@mryenq: do you remember why we deleted this?

Reply:
@jmhooper Currently it's duplicated:

https://github.com/18F/identity-idp/blob/0327095999f3975cd4dcd911ca277f8175a4f27a/app/assets/stylesheets/components/all.scss#L11

https://github.com/18F/identity-idp/blob/0327095999f3975cd4dcd911ca277f8175a4f27a/app/assets/stylesheets/components/all.scss#L23